### PR TITLE
Android Agent 6.1.0 Release Notes

### PR DESCRIPTION
**Pull request by ywang-nr**
### Give us some context

* What problems does this PR solve? - release notes for next android agent

### Are you making a change to site code?
N/A

### CODE REVIEW

**Comment by ywang-nr** [comment]
Hi, @rhetoric101, I would need someone from my team to review my changes, but I couldn't assign reviewers, could you direct me how to do that? Thanks

**Comment by rhetoric101** [comment]
@cthomas-newrelic, can you provide a technical review of these release notes?

**Comment by cthomas-newrelic** [comment]
Final review will occur when agent is released. Until then it is in-progess.

**Review by barbnewrelic — APPROVED** [review]
Looks great, thanks!

**Comment by barbnewrelic** [comment]
Moving this temporarily to Waiting on SME/Blocked until @cthomas-newrelic  has a chance to give final approval, No rush, thanks!

**Comment by ywang-nr** [comment]
1> updated all the related documentations for next android agent 6.1.0 release
2> Note: need to translate JP version of requirements

**Comment by ywang-nr** [comment]
We are good to release this doc when this PR is ready, android agent is released to PROD.

**Review by rhetoric101 — COMMENTED** [review]
@ywang-nr and @cthomas-newrelic, here are a few suggestions.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-610.mdx
Comment (cthomas-newrelic) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-610.mdx:14]: No, should be `6.0.0`, to set the context. It's a short version of what was added to teh config doc.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx
Comment (cthomas-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx:38]: Certainly, adjust for readability and best-practices. Thanks for the feedback!

**Review by cthomas-newrelic — DISMISSED** [review]
Please make recommended changes, then good to go!

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx
Comment (rhetoric101) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx:38]: @ywang-nr and @cthomas-newrelic, I made some formatting and wording changes in the crash reporting doc.

Comment (ywang-nr) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/android-agent-crash-reporting.mdx:38]: awesome! thank you, did you commit your changes? Once you commit and approve, I will merge the PR.

**Review by cthomas-newrelic — APPROVED** [review]
Ship it

**Review by paperclypse — APPROVED** [review]
Looks good to me! I'm merging it in and it will go out with this morning's merge into main.

**Pull request merged**
